internal/execute/tsc.go

```diff
@@ -36,12 +36,15 @@ func executeCommandLineWorker(sys System, cb cbType, commandLine *tsoptions.Pars
 
 	if commandLine.CompilerOptions().Init.IsTrue() ||
 		commandLine.CompilerOptions().Version.IsTrue() ||
-		// commandLine.CompilerOptions().Help != nil ||
-		// commandLine.CompilerOptions().All != nil ||
 		commandLine.CompilerOptions().Watch.IsTrue() && commandLine.CompilerOptions().ListFilesOnly.IsTrue() {
 		return ExitStatusNotImplemented, nil
 	}
 
+	if commandLine.CompilerOptions().Help.IsTrue() || commandLine.CompilerOptions().All.IsTrue() {
```

Comment:
Once the other PR is merged, can you split up the "not implemented" cases above into the "correct" order from Strada? That way we don't fill them in but mistakenly check them in the wrong order.

Reply:
No problem. Rebased and updated.